src/test/java/org/broadinstitute/hellbender/tools/exome/AllelicCountUnitTest.java

```diff
@@ -48,16 +48,4 @@ public void testEstimateAltAlleleFraction() {
         Assert.assertEquals(new AllelicCount(interval, 10, 20).estimateAltAlleleFraction(), 0.6666666, tolerance);
         Assert.assertEquals(new AllelicCount(interval, 10, 90).estimateAltAlleleFraction(), 0.9, tolerance);
     }
-
-    @Test
```

Comment:
It's extraneous.  The "backend" `estimateMinorAlleleFraction` is already tested, and the need to convert to an intermediate `TargetCoverage` no longer exists.